packages/tests/src/nwaku.ts

```diff
@@ -171,6 +176,8 @@ export class Nwaku {
         tcpPort,
         websocketPort,
         ...(args?.peerExchange && { discv5UdpPort }),
+        ...(args?.useFilterv2 &&
+          isGoWaku && { lightClient: true, minRelayPeersToPublish: 0 }),
```

Comment:
Would be good to aim to align go-waku and nwaku behaviour cc @richard-ramos

Reply:
(for some reason i'm not receiving these mentions on my email nor in https://github.com/pulls/mentioned)

Yes! i discussed the other day with @jm-clius if it was an overkill to have this light client functionality flag or not, and I'll change go-waku so it's not necessary. However, the `minRelayPeersToPublish` i'm not sure if it's a good idea to have it set by default to 0. I think it's preferable to receive an error when you attempt to publish a message and have no peers, and I control this with that flag. What do you guys think?

Reply:
In this commit https://github.com/waku-org/go-waku/commit/39e150b1188670688c1ff66d633b458b7e4cb00f I made the light client option to be used only with Legacy Filter. So `lightClient: true` can be removed!